src/hooks/useAuthNavigation.tsx

```diff
@@ -26,5 +27,5 @@ export function useAuthNavigation() {
     ) {
       navigate(PATHS.SIGN_IN)
     }
-  }, [authState])
+  }, [authState, location.pathname])
```

Comment:
@franknoirot this is the actual fix. I'm concerned about side effects coming out of this, so looking for input before settling on the solution